src/models.ts

```diff
@@ -7,21 +7,29 @@ export interface TileModel {
   collections: string[];
   commentCount: number;
   creator?: string;
+  creators: string[];
   dateAdded?: Date; // Date added to public search (software-defined) [from: addeddate]
   dateArchived?: Date; // Date archived (software-defined) item created on archive.org [from: publicdate]
   datePublished?: Date; // Date work published in the world (user-defined) [from: date]
   dateReviewed?: Date; // Date reviewed (user-created) most recent review [from: reviewdate]
   description?: string;
   favCount: number;
   identifier: string;
+  issue?: string;
   itemCount: number;
   mediatype: MediaType;
+  source?: string;
   subjects: string[];
   title: string;
   viewCount: number;
+  volume?: string;
 }
 
-export type CollectionDisplayMode = 'grid' | 'list-compact' | 'list-detail';
+export type CollectionDisplayMode =
+  | 'grid'
+  | 'list-compact'
+  | 'list-detail'
+  | 'list-header';
```

Comment:
Blocker: Can we decouple the tile type from the `CollectionDisplyMode`? This was intended to be used just for the 3 display modes we show the browser in. You could create a `TileType` that is basically this same thing and pass that into the dispatcher instead.